seed/swift-sdk/multi-url-environment/Sources/Resources/Ec2.swift

```diff
@@ -5,7 +5,13 @@ public final class Ec2Client: Sendable {
         self.httpClient = HTTPClient(config: config)
     }
 
-    public func bootInstance(requestOptions: RequestOptions? = nil) async throws -> Any {
-        fatalError("Not implemented.")
+    public func bootInstance(request: Any, requestOptions: RequestOptions? = nil) async throws -> Any {
```

Comment:
nit: not required for now but this endpoint makes a request to a different base url than S3.swift aso ideally you should pass in the base url into `performRequest`